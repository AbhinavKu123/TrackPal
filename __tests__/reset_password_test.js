import React from 'react';
import ResetPassword from '../app/screens/ResetPasswordScreen/resetPasswordScreen.js';

import renderer from 'react-test-renderer';

jest.useFakeTimers();

test('renders correctly', () => {
  const tree = renderer.create(<ResetPassword />).toJSON();
  expect(tree).toMatchSnapshot();
});