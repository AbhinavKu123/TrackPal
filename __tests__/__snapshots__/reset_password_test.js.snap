// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  accessibilityIgnoresInvertColors={true}
  style={
    Object {
      "flex": 1,
      "height": "100%",
      "width": "100%",
    }
  }
>
  <Image
    source={
      Object {
        "testUri": "../../../app/images/bg_image.png",
      }
    }
    style={
      Array [
        Object {
          "bottom": 0,
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
        },
        Object {
          "height": "100%",
          "width": "100%",
        },
        undefined,
      ]
    }
  />
  <View
    style={
      Object {
        "alignItems": "center",
        "flex": 1,
      }
    }
  >
    <RCTScrollView
      style={
        Object {
          "flex": 1,
        }
      }
    >
      <View>
        <View
          style={
            Object {
              "alignItems": "center",
              "flex": 0.5,
              "justifyContent": "flex-start",
              "marginBottom": 30,
              "marginTop": 266.8,
            }
          }
        >
          <Image
            source={
              Object {
                "testUri": "../../../app/images/New2.png",
              }
            }
            style={
              Object {
                "height": 100,
                "width": "100%",
              }
            }
          />
        </View>
        <View
          onLayout={[Function]}
        >
          <View
            style={
              Object {
                "bottom": 0,
              }
            }
          >
            <View
              style={
                Object {
                  "alignItems": "center",
                  "borderRadius": 50,
                  "justifyContent": "flex-start",
                }
              }
            >
              <TextInput
                allowFontScaling={true}
                keyboardType="email-address"
                onChangeText={[Function]}
                placeholder="Email"
                placeholderTextColor="white"
                style={
                  Object {
                    "backgroundColor": "rgba(0,0,0,0.2)",
                    "borderColor": "#fff",
                    "borderRadius": 50,
                    "borderWidth": 1,
                    "color": "white",
                    "height": 60,
                    "marginBottom": 15,
                    "paddingHorizontal": 10,
                    "textAlign": "center",
                    "width": 600,
                  }
                }
                underlineColorAndroid="transparent"
              />
              <View
                accessible={true}
                isTVSelectable={true}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                style={
                  Object {
                    "alignItems": "center",
                    "backgroundColor": "white",
                    "borderRadius": 50,
                    "height": 60,
                    "marginTop": 10,
                    "opacity": 1,
                    "padding": 22,
                    "width": 330,
                  }
                }
              >
                <Text
                  style={
                    Object {
                      "color": "black",
                      "fontSize": 16,
                    }
                  }
                >
                  Reset Password
                </Text>
              </View>
            </View>
          </View>
        </View>
        <View
          style={
            Object {
              "alignItems": "center",
              "flex": 0.5,
              "justifyContent": "flex-start",
              "marginTop": 25,
            }
          }
        >
          <View
            accessible={true}
            isTVSelectable={true}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "opacity": 1,
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "white",
                  "fontSize": 16,
                  "fontWeight": "bold",
                }
              }
            >
              Go Back To Login
            </Text>
          </View>
        </View>
      </View>
    </RCTScrollView>
  </View>
</View>
`;
